cmd/substreams-sink-postgres/run.go

```diff
@@ -35,6 +36,8 @@ var sinkRunCmd = Command(sinkRunE,
 			updates to the cursor will overwrite the module hash in the database.
 		`),
 		)
+		flags.String("rollback-url", "", "url of the rollback service")
+		flags.String("rollback-db-schema", "", "schema rollback service")
```

Comment:
I'm thinking about supporting different rollbacks techniques in the future so I'll probably revisit how we enable the correct "rollback handler" before shipping the feature in an official release.